### ISSUE TITLE
Bleadperl v5.29.8-131-g2abbd513b breaks PERRAD/re-engine-Lua-0.17.tar.gz

### ISSUE THREAD

**Issue by p5pRT** [Perl/perl5#16912]
Migrated from [rt.perl.org#133967](https://rt-archive.perl.org/perl5/Ticket/Display.html?id=133967) (status was 'resolved')

Searchable as RT133967$

**Comment by p5pRT** [comment]
### From @andk

Xref&#8203;: https://rt.cpan.org/Ticket/Display.html?id=128968
\-\- 
andreas
perl perl perl

**Comment by p5pRT** [comment]
### From @hvds

On Sun\, 31 Mar 2019 03&#8203;:25&#8203;:03 \-0700\, andreas\.koenig\.7os6VVqR@&#8203;franz\.ak\.mind\.de wrote&#8203;:
> Xref&#8203;: https://rt.cpan.org/Ticket/Display.html?id=128968

The test is using a variable\-length lookbehind expecting it to die\, for no obvious reason \- it appears to be attempting to verify that re\-engine\-Lua's unimport works\, so perhaps this is something supported by the Lua engine\.

The test will need to change\.

Hugo

**Comment by p5pRT** [comment]
The RT System itself - Status changed from 'new' to 'open'

**Comment by p5pRT** [comment]
### From @eserte

Dana Sun\, 31 Mar 2019 03&#8203;:25&#8203;:03 \-0700\, andreas\.koenig\.7os6VVqR@&#8203;franz\.ak\.mind\.de reče&#8203;:
> Xref&#8203;: https://rt.cpan.org/Ticket/Display.html?id=128968

RURBAN/re\-engine\-PCRE2\-0\.14\.tar\.gz has a very similar test failure&#8203;:

\.\.\.
Variable length lookbehind is experimental in regex; marked by \<\-\- HERE in m/\(?\<=Moose|Mo\)\, \(world\) \<\-\- HERE / at \(eval 66\) line 1\.
\#   Failed test at t/1\-basic\.t line 14\.
\#          got&#8203;: ''
\#     expected&#8203;: undef
\# Looks like you failed 1 test of 6\.
t/1\-basic\.t \.\.\.\.\.\.\.\.\.\.\.\.\.\.\. 
Dubious\, test returned 1 \(wstat 256\, 0x100\)
Failed 1/6 subtests 
\.\.\.

**Comment by p5pRT** [comment]
### From @jkeenan

On Sun\, 07 Apr 2019 20&#8203;:06&#8203;:13 GMT\, slaven@&#8203;rezic\.de wrote&#8203;:
> Dana Sun\, 31 Mar 2019 03&#8203;:25&#8203;:03 \-0700\,
> andreas\.koenig\.7os6VVqR@&#8203;franz\.ak\.mind\.de reče&#8203;:
> > Xref&#8203;: https://rt.cpan.org/Ticket/Display.html?id=128968
> 
> RURBAN/re\-engine\-PCRE2\-0\.14\.tar\.gz has a very similar test failure&#8203;:
> 
> \.\.\.
> Variable length lookbehind is experimental in regex; marked by \<\-\-
> HERE in m/\(?\<=Moose|Mo\)\, \(world\) \<\-\- HERE / at \(eval 66\) line 1\.
> \#   Failed test at t/1\-basic\.t line 14\.
> \#          got&#8203;: ''
> \#     expected&#8203;: undef
> \# Looks like you failed 1 test of 6\.
>  t/1\-basic\.t \.\.\.\.\.\.\.\.\.\.\.\.\.\.\.
> Dubious\, test returned 1 \(wstat 256\, 0x100\)
>  Failed 1/6 subtests
> \.\.\.

I just bisected the RURBAN/re\-engine\-PCRE2 failure to this commit&#8203;:

\#\#\#\#\#
73b95840bb1b55d761ec2dd075d2a8c37fa94bf4 is the first bad commit
commit 73b95840bb1b55d761ec2dd075d2a8c37fa94bf4
Author&#8203;: Karl Williamson \<khw@&#8203;cpan\.org>
Date&#8203;:   Mon Aug 20 18&#8203;:31&#8203;:04 2018 \-0600

&nbsp;    Move \\p\{user\-defined\} to core from utf8\_heavy\.pl
\#\#\#\#\#

\.\.\. which is the same commit where breakage was reported in https://rt.perl.org/Public/Bug/Display.html?id=133860.&nbsp;  So I recommend that this ticket be merged into that one\.

Thank you very much\.
\-\- 
James E Keenan \(jkeenan@&#8203;cpan\.org\)

**Comment by p5pRT** [comment]
### From @jkeenan

On Sun\, 07 Apr 2019 23&#8203;:41&#8203;:33 GMT\, jkeenan wrote&#8203;:
> On Sun\, 07 Apr 2019 20&#8203;:06&#8203;:13 GMT\, slaven@&#8203;rezic\.de wrote&#8203;:
> > Dana Sun\, 31 Mar 2019 03&#8203;:25&#8203;:03 \-0700\,
> > andreas\.koenig\.7os6VVqR@&#8203;franz\.ak\.mind\.de reče&#8203;:
> > > Xref&#8203;: https://rt.cpan.org/Ticket/Display.html?id=128968
> >
> > RURBAN/re\-engine\-PCRE2\-0\.14\.tar\.gz has a very similar test failure&#8203;:
> >
> > \.\.\.
> > Variable length lookbehind is experimental in regex; marked by \<\-\-
> > HERE in m/\(?\<=Moose|Mo\)\, \(world\) \<\-\- HERE / at \(eval 66\) line 1\.
> > \#   Failed test at t/1\-basic\.t line 14\.
> > \#          got&#8203;: ''
> > \#     expected&#8203;: undef
> > \# Looks like you failed 1 test of 6\.
> >  t/1\-basic\.t \.\.\.\.\.\.\.\.\.\.\.\.\.\.\.
> > Dubious\, test returned 1 \(wstat 256\, 0x100\)
> >  Failed 1/6 subtests
> > \.\.\.
> 
> I just bisected the RURBAN/re\-engine\-PCRE2 failure to this commit&#8203;:
> 
> \#\#\#\#\#
> 73b95840bb1b55d761ec2dd075d2a8c37fa94bf4 is the first bad commit
> commit 73b95840bb1b55d761ec2dd075d2a8c37fa94bf4
> Author&#8203;: Karl Williamson \<khw@&#8203;cpan\.org>
> Date&#8203;:   Mon Aug 20 18&#8203;:31&#8203;:04 2018 \-0600
> 
> Move \\p\{user\-defined\} to core from utf8\_heavy\.pl
> \#\#\#\#\#
> 
> \.\.\. which is the same commit where breakage was reported in
> https://rt.perl.org/Public/Bug/Display.html?id=133860.&nbsp;  So I recommend
> that this ticket be merged into that one\.
> 
> Thank you very much\.

Ignore the previous post\, at least temporarily\.  I got confused as to which module I was bisecting for\.

\-\- 
James E Keenan \(jkeenan@&#8203;cpan\.org\)

**Comment by p5pRT** [comment]
### From @iabyn

On Sun\, Apr 07\, 2019 at 04&#8203;:41&#8203;:34PM \-0700\, James E Keenan via RT wrote&#8203;:
> > RURBAN/re\-engine\-PCRE2\-0\.14\.tar\.gz has a very similar test failure&#8203;:

Given that both these distributions have now had new releases which
address the change in perl \(variable\-length look\-behinds now supported\)\,
I propose that this ticket be closed\.

\-\- 
"I used to be with it\, but then they changed what ‘it’ was\, and now what
I’m with isn’t it\. And what’s ‘it’ seems weird and scary to me\."
&nbsp;  \-\- Grandpa Simpson
\(It will happen to you too\.\)

**Comment by p5pRT** [comment]
### From @jkeenan

On Wed\, 24 Apr 2019 14&#8203;:29&#8203;:29 GMT\, davem wrote&#8203;:
> On Sun\, Apr 07\, 2019 at 04&#8203;:41&#8203;:34PM \-0700\, James E Keenan via RT wrote&#8203;:
> > > RURBAN/re\-engine\-PCRE2\-0\.14\.tar\.gz has a very similar test failure&#8203;:
> 
> Given that both these distributions have now had new releases which
> address the change in perl \(variable\-length look\-behinds now supported\)\,
> I propose that this ticket be closed\.
> 

I agree that re\-engine\-Lua now installs satisfactorily\.

However\, I'm getting build\-time errors with re\-engine\-PCRE2 on Linux\.  See cpanm build\.log attached\.

On FreeBSD\-11\.2\, I'm getting what appears to be an indefinite hang in the installation of Alien\-Build\-1\.68\, which is a prerequisite for re\-engine\-PCRE2\.

Thank you very much\.

\-\- 
James E Keenan \(jkeenan@&#8203;cpan\.org\)

**Comment by p5pRT** [comment]
### From @jkeenan

cpanm \(App&#8203;::cpanminus\) 1\.7044 on perl 5\.029010 built for x86\_64\-linux
Work directory is /home/jkeenan/\.cpanm/work/1556119843\.18292
You have make /usr/bin/make
You have /usr/bin/wget
You have /bin/tar&#8203;: tar \(GNU tar\) 1\.29
Copyright \(C\) 2015 Free Software Foundation\, Inc\.
License GPLv3\+&#8203;: GNU GPL version 3 or later \<http&#8203;://gnu\.org/licenses/gpl\.html>\.
This is free software&#8203;: you are free to change and redistribute it\.
There is NO WARRANTY\, to the extent permitted by law\.

Written by John Gilmore and Jay Fenlason\.
You have /usr/bin/unzip
Searching re&#8203;::engine&#8203;::Lua \(\) on cpanmetadb \.\.\.
re&#8203;::engine&#8203;::Lua is up to date\. \(0\.19\)
Searching re&#8203;::engine&#8203;::PCRE2 \(\) on cpanmetadb \.\.\.
\-\-> Working on re&#8203;::engine&#8203;::PCRE2
Fetching http&#8203;://www\.cpan\.org/authors/id/R/RU/RURBAN/re\-engine\-PCRE2\-0\.15\.tar\.gz
\-> OK
Unpacking re\-engine\-PCRE2\-0\.15\.tar\.gz
Entering re\-engine\-PCRE2\-0\.15
Checking configure dependencies from META\.json
Checking if you have Alien&#8203;::PCRE2 0 \.\.\. Yes \(0\.015000\)
Checking if you have ExtUtils&#8203;::MakeMaker 6\.58 \.\.\. Yes \(7\.34\)
Checking if you have Devel&#8203;::CheckLib 1\.00 \.\.\. Yes \(1\.13\)
Configuring re\-engine\-PCRE2\-0\.15
Running Makefile\.PL
Checking if your kit is complete\.\.\.
Looks good
Generating a Unix\-style Makefile
Writing Makefile for re&#8203;::engine&#8203;::PCRE2
Writing MYMETA\.yml and MYMETA\.json
\-> OK
Checking dependencies from MYMETA\.json \.\.\.
Checking if you have Alien&#8203;::PCRE2 0 \.\.\. Yes \(0\.015000\)
Checking if you have Test&#8203;::More 0\.88 \.\.\. Yes \(1\.302162\)
Checking if you have Devel&#8203;::CheckLib 1\.00 \.\.\. Yes \(1\.13\)
Checking if you have Data&#8203;::Dumper 0 \.\.\. Yes \(2\.174\)
Building and testing re\-engine\-PCRE2\-0\.15
cp PCRE2\.pm blib/lib/re/engine/PCRE2\.pm
Running Mkbootstrap for PCRE2 \(\)
chmod 644 "PCRE2\.bs"
"/home/jkeenan/testing/blead/bin/perl" \-MExtUtils&#8203;::Command&#8203;::MM \-e 'cp\_nonempty' \-\- PCRE2\.bs blib/arch/auto/re/engine/PCRE2/PCRE2\.bs 644
"/home/jkeenan/testing/blead/bin/perl" "/home/jkeenan/testing/blead/lib/perl5/5\.29\.10/ExtUtils/xsubpp"  \-typemap '/home/jkeenan/testing/blead/lib/perl5/5\.29\.10/ExtUtils/typemap' \-typemap '/home/jkeenan/\.cpanm/work/1556119843\.18292/re\-engine\-PCRE2\-0\.15/typemap'  PCRE2\.xs > PCRE2\.xsc
mv PCRE2\.xsc PCRE2\.c
cc \-c   \-I/home/jkeenan/testing/blead/lib/perl5/site\_perl/5\.29\.10/x86\_64\-linux/auto/share/dist/Alien\-PCRE2/include \-fwrapv \-fno\-strict\-aliasing \-pipe \-fstack\-protector\-strong \-I/usr/local/include \-D\_LARGEFILE\_SOURCE \-D\_FILE\_OFFSET\_BITS=64 \-O2   \-DVERSION=\\"0\.15\\" \-DXS\_VERSION=\\"0\.15\\" \-fPIC "\-I/home/jkeenan/testing/blead/lib/perl5/5\.29\.10/x86\_64\-linux/CORE"   PCRE2\.c
rm \-f blib/arch/auto/re/engine/PCRE2/PCRE2\.so
LD\_RUN\_PATH="/usr/lib/x86\_64\-linux\-gnu" cc  \-I/home/jkeenan/testing/blead/lib/perl5/site\_perl/5\.29\.10/x86\_64\-linux/auto/share/dist/Alien\-PCRE2/lib \-shared \-O2 \-L/usr/local/lib \-fstack\-protector\-strong  PCRE2\.o  \-o blib/arch/auto/re/engine/PCRE2/PCRE2\.so  \\
&nbsp;   \-lpcre2\-8   \\
&nbsp;  
/usr/bin/ld&#8203;: cannot find \-lpcre2\-8
collect2&#8203;: error&#8203;: ld returned 1 exit status
Makefile&#8203;:518&#8203;: recipe for target 'blib/arch/auto/re/engine/PCRE2/PCRE2\.so' failed
make&#8203;: \*\*\* \[blib/arch/auto/re/engine/PCRE2/PCRE2\.so\] Error 1
\-> FAIL Installing re&#8203;::engine&#8203;::PCRE2 failed\. See /home/jkeenan/\.cpanm/work/1556119843\.18292/build\.log for details\. Retry with \-\-force to force install it\.

**Comment by p5pRT** [comment]
### From @khwilliamson

On Wed\, 24 Apr 2019 08&#8203;:41&#8203;:20 \-0700\, jkeenan wrote&#8203;:
> On Wed\, 24 Apr 2019 14&#8203;:29&#8203;:29 GMT\, davem wrote&#8203;:
> > On Sun\, Apr 07\, 2019 at 04&#8203;:41&#8203;:34PM \-0700\, James E Keenan via RT
> > wrote&#8203;:
> > > > RURBAN/re\-engine\-PCRE2\-0\.14\.tar\.gz has a very similar test
> > > > failure&#8203;:
> >
> > Given that both these distributions have now had new releases which
> > address the change in perl \(variable\-length look\-behinds now
> > supported\)\,
> > I propose that this ticket be closed\.
> >
> 
> I agree that re\-engine\-Lua now installs satisfactorily\.
> 
> However\, I'm getting build\-time errors with re\-engine\-PCRE2 on Linux\.
> See cpanm build\.log attached\.
> 
> On FreeBSD\-11\.2\, I'm getting what appears to be an indefinite hang in
> the installation of Alien\-Build\-1\.68\, which is a prerequisite for re\-
> engine\-PCRE2\.
> 
> Thank you very much\.


Those sound so different from the topic of this ticket\, that I think this should be closed and a new one\(s\) opened for them
\-\- 
Karl Williamson

**Comment by p5pRT** [comment]
### From @jkeenan

On Wed\, 24 Apr 2019 15&#8203;:45&#8203;:20 GMT\, khw wrote&#8203;:
> On Wed\, 24 Apr 2019 08&#8203;:41&#8203;:20 \-0700\, jkeenan wrote&#8203;:
> > On Wed\, 24 Apr 2019 14&#8203;:29&#8203;:29 GMT\, davem wrote&#8203;:
> > > On Sun\, Apr 07\, 2019 at 04&#8203;:41&#8203;:34PM \-0700\, James E Keenan via RT
> > > wrote&#8203;:
> > > > > RURBAN/re\-engine\-PCRE2\-0\.14\.tar\.gz has a very similar test
> > > > > failure&#8203;:
> > >
> > > Given that both these distributions have now had new releases which
> > > address the change in perl \(variable\-length look\-behinds now
> > > supported\)\,
> > > I propose that this ticket be closed\.
> > >
> >
> > I agree that re\-engine\-Lua now installs satisfactorily\.
> >
> > However\, I'm getting build\-time errors with re\-engine\-PCRE2 on Linux\.
> > See cpanm build\.log attached\.
> >
> > On FreeBSD\-11\.2\, I'm getting what appears to be an indefinite hang in
> > the installation of Alien\-Build\-1\.68\, which is a prerequisite for re\-
> > engine\-PCRE2\.
> >
> > Thank you very much\.
> 
> 
> Those sound so different from the topic of this ticket\, that I think
> this should be closed and a new one\(s\) opened for them

I finally managed to get Alien&#8203;::Build to install against blead on FreeBSD\-11\.2\.  I then used cpanm to try to install re&#8203;::engine&#8203;::PCRE2\.  I got test failures\.  But the module's maintainer has already filed a github issue for these failures \(https://github.com/rurban/re-engine-PCRE2/issues/33), to which I just added data\.  So I think re&#8203;::engine&#8203;::PCRE2 has its own problems which need to be sorted out before we can deem it a "pure" BBC situation\.

Should this ticket be closed?  I won't object\.  But at the very least I feel that this ticket should no longer be a 5\.30 blocker\.

Thank you very much\.
Jim Keenan
\-\- 
James E Keenan \(jkeenan@&#8203;cpan\.org\)

**Comment by p5pRT** [comment]
### From @khwilliamson

On Wed\, 24 Apr 2019 09&#8203;:08&#8203;:15 \-0700\, jkeenan wrote&#8203;:
> On Wed\, 24 Apr 2019 15&#8203;:45&#8203;:20 GMT\, khw wrote&#8203;:
> > On Wed\, 24 Apr 2019 08&#8203;:41&#8203;:20 \-0700\, jkeenan wrote&#8203;:
> > > On Wed\, 24 Apr 2019 14&#8203;:29&#8203;:29 GMT\, davem wrote&#8203;:
> > > > On Sun\, Apr 07\, 2019 at 04&#8203;:41&#8203;:34PM \-0700\, James E Keenan via RT
> > > > wrote&#8203;:
> > > > > > RURBAN/re\-engine\-PCRE2\-0\.14\.tar\.gz has a very similar test
> > > > > > failure&#8203;:
> > > >
> > > > Given that both these distributions have now had new releases
> > > > which
> > > > address the change in perl \(variable\-length look\-behinds now
> > > > supported\)\,
> > > > I propose that this ticket be closed\.
> > > >
> > >
> > > I agree that re\-engine\-Lua now installs satisfactorily\.
> > >
> > > However\, I'm getting build\-time errors with re\-engine\-PCRE2 on
> > > Linux\.
> > > See cpanm build\.log attached\.
> > >
> > > On FreeBSD\-11\.2\, I'm getting what appears to be an indefinite hang
> > > in
> > > the installation of Alien\-Build\-1\.68\, which is a prerequisite for
> > > re\-
> > > engine\-PCRE2\.
> > >
> > > Thank you very much\.
> >
> >
> > Those sound so different from the topic of this ticket\, that I think
> > this should be closed and a new one\(s\) opened for them
> 
> I finally managed to get Alien&#8203;::Build to install against blead on
> FreeBSD\-11\.2\.  I then used cpanm to try to install re&#8203;::engine&#8203;::PCRE2\.
> I got test failures\.  But the module's maintainer has already filed a
> github issue for these failures \(https://github.com/rurban/re-engine-
> PCRE2/issues/33\)\, to which I just added data\.  So I think
> re&#8203;::engine&#8203;::PCRE2 has its own problems which need to be sorted out
> before we can deem it a "pure" BBC situation\.
> 
> Should this ticket be closed?  I won't object\.  But at the very least
> I feel that this ticket should no longer be a 5\.30 blocker\.
> 
> Thank you very much\.
> Jim Keenan

Removing from blockers list
\-\- 
Karl Williamson

**Comment by p5pRT** [comment]
### From @khwilliamson

On Wed\, 24 Apr 2019 10&#8203;:22&#8203;:07 \-0700\, khw wrote&#8203;:
> On Wed\, 24 Apr 2019 09&#8203;:08&#8203;:15 \-0700\, jkeenan wrote&#8203;:
> > On Wed\, 24 Apr 2019 15&#8203;:45&#8203;:20 GMT\, khw wrote&#8203;:
> > > On Wed\, 24 Apr 2019 08&#8203;:41&#8203;:20 \-0700\, jkeenan wrote&#8203;:
> > > > On Wed\, 24 Apr 2019 14&#8203;:29&#8203;:29 GMT\, davem wrote&#8203;:
> > > > > On Sun\, Apr 07\, 2019 at 04&#8203;:41&#8203;:34PM \-0700\, James E Keenan via RT
> > > > > wrote&#8203;:
> > > > > > > RURBAN/re\-engine\-PCRE2\-0\.14\.tar\.gz has a very similar test
> > > > > > > failure&#8203;:
> > > > >
> > > > > Given that both these distributions have now had new releases
> > > > > which
> > > > > address the change in perl \(variable\-length look\-behinds now
> > > > > supported\)\,
> > > > > I propose that this ticket be closed\.
> > > > >
> > > >
> > > > I agree that re\-engine\-Lua now installs satisfactorily\.
> > > >
> > > > However\, I'm getting build\-time errors with re\-engine\-PCRE2 on
> > > > Linux\.
> > > > See cpanm build\.log attached\.
> > > >
> > > > On FreeBSD\-11\.2\, I'm getting what appears to be an indefinite hang
> > > > in
> > > > the installation of Alien\-Build\-1\.68\, which is a prerequisite for
> > > > re\-
> > > > engine\-PCRE2\.
> > > >
> > > > Thank you very much\.
> > >
> > >
> > > Those sound so different from the topic of this ticket\, that I think
> > > this should be closed and a new one\(s\) opened for them
> > 
> > I finally managed to get Alien&#8203;::Build to install against blead on
> > FreeBSD\-11\.2\.  I then used cpanm to try to install re&#8203;::engine&#8203;::PCRE2\.
> > I got test failures\.  But the module's maintainer has already filed a
> > github issue for these failures \(https://github.com/rurban/re-engine-
> > PCRE2/issues/33\)\, to which I just added data\.  So I think
> > re&#8203;::engine&#8203;::PCRE2 has its own problems which need to be sorted out
> > before we can deem it a "pure" BBC situation\.
> > 
> > Should this ticket be closed?  I won't object\.  But at the very least
> > I feel that this ticket should no longer be a 5\.30 blocker\.
> > 
> > Thank you very much\.
> > Jim Keenan
> 
> Removing from blockers list

I'm thinking this can now be closed\.  Any objections?
\-\- 
Karl Williamson

**Comment by p5pRT** [comment]
### From @jkeenan

On Thu\, 27 Jun 2019 16&#8203;:49&#8203;:34 GMT\, khw wrote&#8203;:
> On Wed\, 24 Apr 2019 10&#8203;:22&#8203;:07 \-0700\, khw wrote&#8203;:
> > On Wed\, 24 Apr 2019 09&#8203;:08&#8203;:15 \-0700\, jkeenan wrote&#8203;:
> > > On Wed\, 24 Apr 2019 15&#8203;:45&#8203;:20 GMT\, khw wrote&#8203;:
> > > > On Wed\, 24 Apr 2019 08&#8203;:41&#8203;:20 \-0700\, jkeenan wrote&#8203;:
> > > > > On Wed\, 24 Apr 2019 14&#8203;:29&#8203;:29 GMT\, davem wrote&#8203;:
> > > > > > On Sun\, Apr 07\, 2019 at 04&#8203;:41&#8203;:34PM \-0700\, James E Keenan via RT
> > > > > > wrote&#8203;:
> > > > > > > > RURBAN/re\-engine\-PCRE2\-0\.14\.tar\.gz has a very similar test
> > > > > > > > failure&#8203;:
> > > > > >
> > > > > > Given that both these distributions have now had new releases
> > > > > > which
> > > > > > address the change in perl \(variable\-length look\-behinds now
> > > > > > supported\)\,
> > > > > > I propose that this ticket be closed\.
> > > > > >
> > > > >
> > > > > I agree that re\-engine\-Lua now installs satisfactorily\.
> > > > >
> > > > > However\, I'm getting build\-time errors with re\-engine\-PCRE2 on
> > > > > Linux\.
> > > > > See cpanm build\.log attached\.
> > > > >
> > > > > On FreeBSD\-11\.2\, I'm getting what appears to be an indefinite hang
> > > > > in
> > > > > the installation of Alien\-Build\-1\.68\, which is a prerequisite for
> > > > > re\-
> > > > > engine\-PCRE2\.
> > > > >
> > > > > Thank you very much\.
> > > >
> > > >
> > > > Those sound so different from the topic of this ticket\, that I think
> > > > this should be closed and a new one\(s\) opened for them
> > > 
> > > I finally managed to get Alien&#8203;::Build to install against blead on
> > > FreeBSD\-11\.2\.  I then used cpanm to try to install re&#8203;::engine&#8203;::PCRE2\.
> > > I got test failures\.  But the module's maintainer has already filed a
> > > github issue for these failures \(https://github.com/rurban/re-engine-
> > > PCRE2/issues/33\)\, to which I just added data\.  So I think
> > > re&#8203;::engine&#8203;::PCRE2 has its own problems which need to be sorted out
> > > before we can deem it a "pure" BBC situation\.
> > > 
> > > Should this ticket be closed?  I won't object\.  But at the very least
> > > I feel that this ticket should no longer be a 5\.30 blocker\.
> > > 
> > > Thank you very much\.
> > > Jim Keenan
> > 
> > Removing from blockers list
> 
> I'm thinking this can now be closed\.  Any objections?

No objection\.  Marking Resolved\.

\-\- 
James E Keenan \(jkeenan@&#8203;cpan\.org\)

**Comment by p5pRT** [comment]
@jkeenan - Status changed from 'open' to 'resolved'
